### ISSUE TITLE
Month-Year Picker Issue on ASP.NET MVC 4 Web App

### ISSUE THREAD

**Issue by corix010** [Eonasdan/bootstrap-datetimepicker#2061]
I have an existing ASP.NET MVC 4 web app and trying to use Bootstrap 3 Datepicker for my "month/year" field like the one in demo here:
https://eonasdan.github.io/bootstrap-datetimepicker/#min-view-mode

In that demo, it only allows users to select "year" and "month". Not "days". I want the same functionality in my app. However, in my app, it works fine on first click but on succeeding clicks it shows the "days" selection view. Setting the viewMode to 'years' doesn't fix the problem.

My model is as follows:
```
    public class FilterViewModel
    {
        public DateTime? StartMonthYear { get; set; }
        ...
```

My view is as follows:
```
    <div class="col-sm-3">
        @Html.TextBoxFor(model => model.StartMonthYear, new { @class = "month-picker" })
        ...
    <script type="text/javascript">
        $(function () {
            $('.month-picker').datetimepicker({
                viewMode: 'months',
                format: 'MM/YYYY',
                useCurrent: false
            });
        });
    </script>
```

I installed Bootstrap 3 datepicker via Nuget:

- PM> Install-Package Bootstrap.v3.Datetimepicker
- PM> Install-Package Bootstrap.v3.Datetimepicker.CSS

Packages installed:

- bootstrap.3.3.0
- Bootstrap.Less.3.3.5
- Bootstrap.v3.Datetimepicker.4.17.45
- Bootstrap.v3.Datetimepicker.CSS.4.17.45
- Twitter.Bootstrap.Sass.3.3.7
- jQuery.1.9.0
- jQuery.UI.Combined.1.12.1
- jQuery.Validation.1.16.0

**Comment by William-H-M** [comment]
Hi, this is a duplicate of 
#1854 
#1894 
#1959 
and there's a quick fix at #1897

**Comment by corix010** [comment]
Thanks for the links. #1894 provided me with a good work-around. However, the fixes suggested are just work-arounds not an actual fix for the said issue.

Another suggestion is installing the latest version. However, Nuget would only let me install version up to 4.17.45 only.

**Comment by William-H-M** [comment]
Hi, yeah Nuget is usually not updated I advise you to download it from this repository and Include it or just copy and paste the code on where Nuget installed the plugin

**Comment by corix010** [comment]
Thanks William. I was able to install the latest version manually. I can confirm the bug has been fixed with the latest version (4.17.47). The work-arounds/fixes are no longer needed.

**Comment by corix010** [comment]
The bug has been fixed with the latest version (4.17.47).
